packages/python3.rb

```diff
@@ -85,20 +85,22 @@ def self.build
     # test_urllib, test_urllib2, test_urllib2_localnet.
     # So, modifying environment variable to make pass tests.
 
-    FileUtils.mkdir_p 'builddir'
-    Dir.chdir 'builddir' do
-      system CREW_ENV_OPTIONS_HASH.transform_values { |v| "#{v} #{@cppflags}" }, "../configure #{CREW_OPTIONS} \
-        --with-computed-gotos \
-        --enable-loadable-sqlite-extensions \
-        --with-ensurepip \
-        --enable-optimizations \
-        --with-platlibdir='lib#{CREW_LIB_SUFFIX}' \
-        --with-system-ffi \
-        --with-system-expat \
-        --with-system-libmpdec \
-        --with-libc= \
-        --enable-shared"
-      system 'mold -run make'
+    unless Dir.exist?('builddir')
```

Comment:
Why is this logic needed?

Reply:
When working with a cached build where I am playing with tweaking the install section, this allows one to iterate without having to do a complete rebuild.